onboarding/src/Components/EmployeeProfile/EmployeeProfileUser.js

```diff
@@ -2,66 +2,89 @@ import React from "react";
 import { Link } from "react-router-dom";
 
 
-function EmployeeProfileUser(props) {
+function EmployeeProfileUser({ user, goBackToMainProfilePage }) {
     let avatar = "/onboarding/static/images/unknown-profile.jpg";
-    if(props.user.avatar && props.user.avatar.length > 1)
-        avatar = props.user.avatar;
+    if(user.avatar && user.avatar.length > 1)
+        avatar = user.avatar;
```

Comment:
I'm thinking :thinking: 
how important would be to validate `avatar` here also like:
https://github.com/binawork/OnlineOnboarding/pull/174/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&hide-deleted-files=true#r569395358
User data comes from `onboarding/src/Components/EmployeeProfile/EmployeeProfilePage.js`
and it could be checked there if not here.